autoarray/inversion/regularization/adaptive_brightness.py

```diff
@@ -147,40 +145,22 @@ def __init__(
     def regularization_matrix_from(self, mapper) -> np.ndarray:

```

Comment:
There was unecessary code duplication here, so I moved the old code to `regularization_util`.